Eevee.Sleep.Bot/Handlers/InteractionHandler.cs

```diff
@@ -116,4 +117,19 @@ await interaction
                 throw new ArgumentException($"Unhandled modal ID: {modalId}");
         }
     }
+
+    private static async Task OnButtonClicked(SocketMessageComponent component) {
+        var info = ButtonInteractionInfoSerializer.Deserialize(component.Data.CustomId);
+
+        switch (info?.ButtonId) {
+            case ButtonId.RoleChanger:
+                await component.RespondAsync("Your request has been accepted!", ephemeral: true);
+                if (component.User is not null && component.User is SocketGuildUser user) {
+                    await ButtonClickedHandler.ChangeRoleButtonClicked(info, user);
+                }
+                break;
+            default:
+                throw new ArgumentException($"Unhandled button ID: {info?.ButtonId}");
```

Comment:
Make `info?.ButtonId` a single variable and call it here and in `switch` for easier refactoring in the future if needed.